tests/unit_node/http_test.ts

```diff
@@ -1045,11 +1045,15 @@ Deno.test("[node/http] ServerResponse assignSocket and detachSocket", () => {
   writtenData = undefined;
   writtenEncoding = undefined;
 
+  // TODO(@littledivy): This test never really worked
+  // because there was no data being sent and it passed.
+  //
   // @ts-ignore it's a socket mock
-  res.detachSocket(socket);
```

Comment:
Is this test effected by the changes made in http.ts?

Reply:
yes

Reply:
We have to soon address this issue. It might break `serverless-http` and `serverless-express` https://github.com/denoland/deno/issues/22630

Reply:
`detachSocket` is broken so i doubt it was working as expected. I've opened https://github.com/denoland/deno/issues/24429